src/query/functions/src/scalars/array.rs

```diff
@@ -61,12 +64,25 @@ use itertools::Itertools;
 use siphasher::sip128::Hasher128;
 use siphasher::sip128::SipHasher24;
 
+use crate::aggregates::eval_aggr;
+use crate::AggregateFunctionFactory;
+
+const ARRAY_AGGREGATE_FUNCTIONS: &[(&str, &str); 5] = &[
+    ("array_avg", "avg"),
```

Comment:
const ARRAY_AGGREGATE_FUNCTIONS =  ["avg", "min", "max", ...];

and use `concat!("arr_", agg)`